File: Management/AdminApi/AdminApi/Models/AggregateStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Microsoft.Azure.Amqp.Framing;
using Newtonsoft.Json;

namespace AdminApi.Models
{
    public class AggregateStats
    {
        [JsonProperty(PropertyName = "id")]
        public string id { get; set; }
        public int? windowunit { get; set; }
        public string windowsize { get; set; }
        public double? avg { get; set; }
        public int? min { get; set; }
        public int? max { get; set; }
        public int? sum { get; set; }
        public DateTime? windowendtime { get; set; }
        public override string ToString()
        {
            return JsonConvert.SerializeObject(this);
        }
    }
}

File: Management/AdminUI_Simple/src/app/model/AggregateStats.ts

export class AggregateStats {
    constructor(
        public id: string,
        public windowunit: number,
        public windowsize: string,
        public median: number,
        public avg: number,
        public min: number,
        public max: number,
        public sum: number,
        public windowendtime:Date,
         ) {
    }
}
